src/widget.ts

```diff
@@ -56,22 +86,38 @@ export class VegaModel extends DOMWidgetModel {
 
 export class VegaWidget extends DOMWidgetView {
   result?: Result;
+  isUpdated: boolean = false;
```

Comment:
Can you add a comment what isUpdated means? Does it mean the widget is up to date (then I would suggest `isUpToDate`) or that the widget has ever been updated (I would suggest `hasBeenUpdated` or something like that)?